riscv/insns/vfadd_vf.h

```diff
@@ -0,0 +1,13 @@
+// vfadd.vf vd, vs2, rs1
+VI_VFP_VF_LOOP
+({
+  switch(P.VU.vsew) {
+  case e32:
+    vd = f32_add(rs1, vs2);
+    break;
+  case e16:
+  case e8:
+  default:
+    softfloat_exceptionFlags = 1;
```

Comment:
I think this should be `require(0);` (i.e., throw an illegal instruction exception), rather than setting the FP exception flags.  Same comment for all of these files.

Reply:
Got it.